console/build.sbt

```diff
@@ -7,6 +7,7 @@ libraryDependencies ++= Seq(
   "com.github.scopt"     %% "scopt"         % "3.7.1",
   "com.github.pathikrit" %% "better-files"  % "3.8.0",
   "org.apache.commons"   % "commons-lang3"  % "3.8",
+  "com.typesafe.play" %% "play-json" % "2.7.4",
```

Comment:
I would suggest using `circe` in place of `play-json` - it's the de-facto Scala library for JSON serialization and manipulation.